array.c

```diff
@@ -2710,11 +2823,12 @@ rb_ary_length(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.empty?   -> true or false
- *
- *  Returns +true+ if +self+ contains no elements.
+ *    array.empty?  -> true or false
  *
- *     [].empty?   #=> true
+ *  Returns +true+ if the count of elements in the array is zero,
+ *  +false+ otherwise:
+ *    [].empty? # => true
```

Comment:
Not sure, but it will be nice to have a example where an array with only a nil element is still counted, as in:

```ruby
[nil].empty? # => true
```

Reply:
From irb:

[nil].empty?                                                                                                                                            # => false

Reply:
Sorry, it was a brain fart (and that's why important the documentation).

Reply:
That's why I crank all these examples through irb!  And I appreciate your watchful eye.